### PR TITLE
Update page_creation.rst

### PR DESCRIPTION
**Pull request by KiberMath**
this can help if first page is not working

### CODE REVIEW

File: book/page_creation.rst [hunk 1]
```diff
@@ -73,6 +73,12 @@ Before diving into this, test it out!
     replace ``http://localhost:8000`` with your host name - like
     ``http://symfony.dev/app_dev.php/lucky/number``.
 
+.. tip::
+    if you cant see your page try to clear cache using command
+    ``php bin/console  cache:clear --env=prod``
```
Comment (xabbuh) [inline book/page_creation.rst]: It looks strange that clearing the cache for the `prod` environment should help with your `dev` environment.

**Comment by WouterJ** [comment]
@KiberMath can you please share some more details about how this worked for you and the error you got before running this command?

I think this may be caused by a confusion. If you use Apache, routes will be run through the prod env. by default because of the `.htaccess` file in [the Standard Edition](https://github.com/symfony/symfony-standard/blob/master/web/.htaccess#L57). You have to change this on your developing machine.

**Comment by KiberMath** [comment]
@WouterJ

yes, I'm using Apache in my development invironment. And that was exactly that situation you've described. That confusion was realy painfull. I was in situation where i cant understand why my code does not working and not showing me any errors.
Some one should resolve this problem and make some tips in documentation about it.
This confusion can forse some beginers on Apache to suicide.

File: book/page_creation.rst [hunk 2]
```diff
@@ -73,6 +73,12 @@ Before diving into this, test it out!
     replace ``http://localhost:8000`` with your host name - like
     ``http://symfony.dev/app_dev.php/lucky/number``.
 
+.. tip::
+    if you cant see your page try to clear cache using command
```
Comment (OskarStark) [inline book/page_creation.rst]: `if` -> `If`

**Comment by WouterJ** [comment]
I'm closing this PR, as it turns out the suggested changes are not correct. I've also created an issue to not forget about this: https://github.com/symfony/symfony-docs/issues/6731

**Pull request closed without merging**